models/dim_dbt__current_models.sql

```diff
@@ -22,9 +22,7 @@ model_executions as (
 -- Get the most recent comile run
 latest_compile as (
 
-    select
-        command_invocation_id,
-        dbt_cloud_run_id
+    select artifact_run_id

```

Comment:
Even if we're joining on this new key, can we keep the `command_invocation_id` and `dbt_cloud_run_id` in these models? Both are potentially useful to users.

Reply:
Oh nvm they weren't in the model output anyway.